spec/requests/api/braintree_spec.rb

```diff
@@ -13,7 +15,7 @@
   end
 
   def post_transaction(opts = {})
-    post '/api/braintree/transaction', {
+    post "/api/braintree/pages/#{page.id}/transaction", {
```

Comment:
yeah the value is available in liquid, so it can get passed to the javascript. you shouldn't have any trouble going in to change, though you're going to have to change some teaspoon tests. good chance to get familiar with it in a low stress way :P